src/Workspaces/Core/Portable/Log/FunctionId.cs

```diff
@@ -78,7 +78,7 @@ internal enum FunctionId
         Workspace_ApplyChanges,
         Workspace_TryGetDocument,
         Workspace_TryGetDocumentFromInProgressSolution,
-        Workspace_Solution_LinkedFileDiffMergingSession,
+        Workspace_Solution_LinkedFileDiffMergingSession, // Not in use
```

Comment:
Sorry for late reply David.

Added some comments

From: David Poeschl [mailto:notifications@github.com]
Sent: Monday, May 25, 2015 12:41 PM
To: dotnet/roslyn
Cc: HeeJae Chang
Subject: Re: [roslyn] Draft/Ignore: Reporting aggregated telemetry on solution close (#3073)


In src/Workspaces/Core/Portable/Log/FunctionId.cs<https://github.com/dotnet/roslyn/pull/3073#discussion_r30992557>:

> @@ -78,7 +78,7 @@ internal enum FunctionId

>          Workspace_ApplyChanges,

>          Workspace_TryGetDocument,

>          Workspace_TryGetDocumentFromInProgressSolution,

> -        Workspace_Solution_LinkedFileDiffMergingSession,

> +        Workspace_Solution_LinkedFileDiffMergingSession, // Not in use

@heejaechang<https://github.com/heejaechang> I have to leave this one in the list to preserve numbering, right?

—
Reply to this email directly or view it on GitHub<https://github.com/dotnet/roslyn/pull/3073/files#r30992557>.